pkgs/build-support/gcc-wrapper/gcc-wrapper.sh

```diff
@@ -79,6 +79,18 @@ if [ "$NIX_ENFORCE_PURITY" = 1 -a -n "$NIX_STORE" ]; then
     params=("${rest[@]}")
 fi
 
+if test -n "@libcxx@"; then
```

Comment:
I don't know, but perhaps @copumpkin or @joelteon do, I think they were the ones who added `libc++`.